contrib/epee/include/net/http_protocol_handler.inl

```diff
@@ -268,7 +269,7 @@ namespace net_utils
 					m_is_stop_handling = true;
 					if(m_cache.size() > HTTP_MAX_URI_LEN)
 					{
-						LOG_ERROR("simple_http_connection_handler::handle_buff_out: Too long URI line");
+						LOG_ERROR("simple_http_connection_handler::handle_buff_out: Too long URI line [" << this->m_conn_context << "]");
```

Comment:
Please use the existing macros for this, such as LOG_ERROR_CC. That way the output's all consistent (and it's easier to write too).

Reply:
Fixed